code/__DEFINES/maths.dm

```diff
@@ -33,6 +33,13 @@
 // round() acts like floor(x, 1) by default but can't handle other values
 #define FLOOR(x, y) ( round((x) / (y)) * (y) )
 
+//clamps which is used for reactors
```

Comment:
I'm gonna be serious here, we do not need a helper for 513 versions, i don't even know if cit would run in it.

Reply:
fair, let me try updating since I think I was still on 513 and I added that so it would compile.

Reply:
If you want some help deciding, for me the last version that's working well is 514.1584